src/features/recovery/RecoveryManager.ts

```diff
@@ -89,7 +89,7 @@ export class RecoveryManager {
         const startTime = stats.birthtimeMs;
         const endTime = stats.mtimeMs;
 
-        return Math.round((endTime - startTime) / 1000);
+        return Math.round((endTime - startTime) / 1000) + 10;
```

Comment:
If you know that sampling rate then you can convert the byte size to time I think

Reply:
@tsvetelinpetrov please implement (if you're going to) on the `feat/recover-on-crash` branch